recovery/elliptics_recovery/monitor.py

```diff
@@ -198,6 +199,9 @@ def data_thread(self):
 
                 # Use different handling for different stat flavours
                 flavour = data[1]
+                if flavour == StatsProxy.SHUTDOWN:
+                    self.log.info('Shutdown monitor')
+                    self.__shutdown_request = True
                 if flavour == StatsProxy.COUNTER:
```

Comment:
`elif`?
Or you want to catch RuntimeError?

Reply:
It was lost, thanks.